### PR TITLE
Issue 1

### PR DESCRIPTION
**Pull request by limmy13**
I've created a very basic homepage wireframe inside the homepage folder that includes index.html and styles.css files. Also wanted to be fun and incorporate some fonts. (I can send the group the fonts in case you need them on your computer.)

### CODE REVIEW

**Comment by limmy13** [comment]
Don't forget to git pull after the branches have been merged!

**Review by Tam-Leonard — APPROVED** [review]

File: incodables-front-end/content-vibes-front-end/src/Homepage/index.html
Comment (jojopenguin191) [inline incodables-front-end/content-vibes-front-end/src/Homepage/index.html:1]: What's with the cheese thing?

Comment (jojopenguin191) [inline incodables-front-end/content-vibes-front-end/src/Homepage/index.html:1]: 1) What is the paragraph about cheese for? 
2) I restructured it a bit and added the closing html tag

<!DOCTYPE html>
<html>
<head>
    <meta charset="UTF-8">
    <title>Homepage</title>
    <link rel="stylesheet" type="text/css" href="./styles.css">
</head>
   
<body>
    <div class="header">
       <h1>RPG VIBES</h1> 
    </div>
    <div class="login">
        <h2>Login</h2>
        
    </div>
    <div class="registration">
        <h2>Registration</h2>
        
    </div>
    <div class="about-us">
        <h2>About Us</h2>
        <p>I love cheese, especially cheese on toast taleggio. Brie cheddar chalk and cheese everyone loves camembert de normandie cottage cheese queso cottage cheese. Lancashire cheese and wine croque monsieur cottage cheese caerphilly cream cheese halloumi when the cheese comes out everybody's happy. Red leicester.

        </p>
    </div>
</body>
</html>

Comment (jojopenguin191) [inline incodables-front-end/content-vibes-front-end/src/Homepage/index.html:1]: You need the closing h t m l tag at the bottom.

Also what's with the cheese thing?

**Review by Tam-Leonard — APPROVED** [review]

**Review by jojopenguin191 — APPROVED** [review]

**Pull request merged**